.github/workflows/build-binaries.yml

```diff
@@ -39,7 +39,7 @@ jobs:
         with:
           node-version: '16'
       - name: Setup
-        run: npm run setup
+        run: npm i
       - name: Get tag
         run: node ./scripts/get-tag.js # ToDo (PRE_NX): replace by some jq magic
```

Comment:
It's normal that I have a ToDo here: I scoped out the build-binaries to avoid adding even more stuff in the chunky #632